### DISCUSSION TITLE
Search doesn't work

### DISCUSSION THREAD

**Discussion by ghnp5** [valinet/ExplorerPatcher#1112] [General]
Hey!
Thanks very much for your latest update, 2 hours ago.
That seems to have fixed a few things (or most anyway) that broke since the latest Windows update.

However, it seems that Search is still not working for me:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/57591332/226122104-c05405ec-3557-4add-ba71-8326e618f0e7.png">

I can't click on the Search bar, and it doesn't work either if I start typing.

I'm surprised no one seems to have reported this yet, so I wonder if it's just me?

Thank you very much!

**Comment by valinet** [comment]
It's been reported: https://github.com/valinet/ExplorerPatcher/issues/1109

But what is there to report? I cannot reproduce it. On my test VM, it works. UWP stuff is fucked up a lot of the times, by design. Try to restart the computer.

**Reply by ghnp5** [reply]
Oh... I searched under Discussions, not Issues. Thanks.
I'll try to restart Windows, soon. :)

**Reply by ghnp5** [reply]
Thanks. Restarting Windows resolved this.
